runtime/perfetto/db.go

```diff
@@ -112,34 +102,32 @@ func Open(ctx context.Context, fname string) (*DB, error) {
 	}
 	db.SetMaxOpenConns(1)
 
-	// Initialize the replica number cache.
-	const cacheSize = 65536
-	cache, err := lru.New[replicaCacheKey, int](cacheSize)
-	if err != nil {
-		return nil, err
-	}
-
 	t := &DB{
-		fname:           fname,
-		db:              db,
-		replicaNumCache: cache,
+		fname: fname,
+		db:    db,
 	}
 
 	const initTables = `
--- Trace data.
-CREATE TABLE IF NOT EXISTS traces (
+-- Trace span data.
+CREATE TABLE IF NOT EXISTS spans (
+	id INTEGER PRIMARY KEY AUTOINCREMENT,
 	app TEXT NOT NULL,
 	version TEXT NOT NULL,
-	events TEXT NOT NULL
+	name TEXT,
+	trace_id TEXT,
+	span_id TEXT,
+	parent_span_id TEXT,
+	start_time_unix_us INTEGER,
+	end_time_unix_us INTEGER,
+	encoded_span TEXT
 );
 
--- Map from a weavelet id to a replica number.
-CREATE TABLE IF NOT EXISTS replica_num (
-	app TEXT NOT NULL,
-	version TEXT NOT NULL,
-	weavelet_id TEXT NOT NULL,
-	num INTEGER NOT NULL,
-	PRIMARY KEY(app,version,weavelet_id)
+-- Trace span attributes.
+CREATE TABLE IF NOT EXISTS span_attributes (
+	id INTEGER NOT NULL,
```

Comment:
In my upcoming change, I'd like to use attributes while querying. In particular, I'd like the HTTP InstrumentHandler to add attributes that mark traces with errors, so that we can query them.

If used by querying, I need the attributes table to be keyed to the traces table.